components/mailgun/sources/new-bounce/new-bounce.js

```diff
@@ -0,0 +1,50 @@
+const common = require("../common-webhook");
+const { mailgun } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-bounce",
+  name: "New bounce",
+  description:
+    "Emit an event when the email recipient who could not be reached.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    domain: { propDefinition: [mailgun, "domain"] },
+  },
+  methods: {
+    ...common.methods,
+    getEventName() {
+      return ["bounce"];
+    },
+    getEventType() {
+      return "bounced";
+    },
+    generateMeta(eventPayload) {
+      const ts = eventPayload.timestamp;
+      return {
+        id: `${eventPayload["X-Mailgun-Sid"]}${ts}`,
+        summary: `New Bounce on message id: ${eventPayload["Message-Id"]} by ${eventPayload.recipient}`,
```

Comment:
In general I like a `summary` that's short and doesn't duplicate any information. The user created a New Bounce source, so we can leave out "New Bounce on message ID". I think the most useful information here is the `recipient`, so I would actually only include that in this case and also remove the `Message-Id`.

Reply:
ok noted. i changed to `summary: eventPayload.recipient,`